dawn/CMakeLists.txt

```diff
@@ -142,6 +142,8 @@ if(NOT Protobuf_FOUND)
   include(FetchProtobuf)
 endif()
 
+find_package(Clang 10.0.0 EXACT REQUIRED)
+
```

Comment:
EXACT version 10.0.0

Checked if this is supported in Ubuntu 20, 18 and spack. All yes.